ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/io/PartitionSerializer.java

```diff
@@ -59,6 +61,11 @@ public void writeTo(JsonWriter writer, ReplicationSpec additionalPropertiesProvi
                   additionalPropertiesProvider.getCurrentReplicationState());
         }
       }
+
+      if (parameters != null && parameters.containsKey(COLUMN_STATS_ACCURATE)) {
```

Comment:
There is already parameters != null check above in this method. Can we move this code there itself?